findbugs-exclude.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!--
-  ~ Copyright (c) 2016 Goldman Sachs and others.
+  ~ Copyright (c) 2016 Goldman Sachs.
```

Comment:
Thanks, fixing this manually. My IDE made this change based on the configuration in `.idea/copyright/Eclipse.xml` so we'll need to figure out how to prevent this from happening automatically.